src/util/api.js

```diff
@@ -0,0 +1,20 @@
+/* eslint no-console: 0 */
+import { get, getAll } from './http'
+import handleFilters from './filters'
+
+const API = '/api'
+const ENDPOINTS = {
+  incidents: `${API}/incidents`,
+}
+
+const getAllIncidents = params => {
+  const processed = handleFilters(params)
+  return getAll(ENDPOINTS.incidents, processed)
+}
+
+const getIncidents = params => {
+  const processed = handleFilters(params)
+  return get(ENDPOINTS.incidents, processed)
+}
+
+export default { getAllIncidents, getIncidents }
```

Comment:
My understanding is that this line means export an object that has two methods as its keys. So this enables the importer to name the object whatever they want, or just pull out the individual methods they need. For example, it seems like both of these would be possible using this method:

```
import { getAllIncidents } from './api'      // if you just want one method
import api from './api'      // is this object { getAllIncidents(), getIncidents() }
import somethingWeird from './api'       // OR name it something else
```

I think this provides for more flexibility for people importing the file than using a named export which they would also have to use.